packages/mobile/src/components/navigation-container/NavigationContainer.tsx

```diff
@@ -14,85 +18,98 @@ import { navigationThemes } from './navigationThemes'
-          profile: {
-            screens: {
-              ProfileStack: {
-                screens: {
-                  Tracks: '*/tracks',
-                  Albums: '*/albums',
-                  Playlists: '*/playlists',
-                  Reposts: '*/reposts',
-                  Collectibles: '*/collectibles/*'
-                }
-              }
-            }
-          }
-        }
-      }
-    }
-  }
-}
-
 /**
  * NavigationContainer contains the react-navigation context
  * and configures linking
  */
 const NavigationContainer = ({ children }: Props) => {
   const { theme, isSystemDarkMode } = useContext(ThemeContext)
+  const pushRouteWeb = usePushRouteWeb()
+  const account = useSelectorWeb(getAccountUser)
 
   const navigationTheme =
     theme === 'auto' ? (isSystemDarkMode ? 'dark' : 'default') : theme
 
+  const linking: LinkingOptions<RootScreenParamList> = {
+    prefixes: ['https://audius.co', 'https://staging.audius.co'],
+    // configuration for matching screens with paths
+    config: {
+      screens: {
+        App: {
+          screens: {
+            MainStack: {
+              initialRouteName: 'feed',
+              screens: {
+                feed: {
+                  initialRouteName: 'Feed',
+                  screens: {
+                    Feed: '/feed',
+                    Collection: '*/playlist/*',
+                    Track: '*/*',
+                    Profile: ':handle'
+                  }
+                },
+                trending: {
+                  initialRouteName: 'Trending',
+                  screens: {
+                    Trending: 'trending'
+                  }
+                },
+                explore: {
+                  initialRouteName: 'Explore',
+                  screens: {
+                    Explore: 'explore',
+                    TrendingPlaylists: 'explore/playlists',
+                    TrendingUnderground: 'explore/underground',
+                    LetThemDJ: 'explore/let-them-dj',
+                    TopAlbums: 'explore/top-albums',
+                    UnderTheRadar: 'explore/under-the-radar',
+                    BestNewReleases: 'explore/best-new-releases',
+                    Remixables: 'explore/remixables',
+                    MostLoved: 'explore/most-loved',
+                    FeelingLucky: 'explore/feeling-lucky',
+                    HeavyRotation: 'explore/heavy-rotation',
+                    ChillPlaylists: 'explore/chill',
+                    IntensePlaylists: 'explore/intense',
+                    IntimatePlaylists: 'explore/intimate',
+                    ProvokingPlaylists: 'explore/provoking',
+                    UpbeatPlaylists: 'explore/upbeat'
+                  }
+                },
+                favorites: {
+                  screens: {
+                    Favorites: 'favorites'
+                  }
+                },
+                profile: {
+                  screens: {
+                    UserProfile: 'profile'
```

Comment:
profile allows /profile/tracks, profile/albums, etc. FYI!

Reply:
Good catch, just got that situated as well!